### PR TITLE
config_tools: fix load board issue

### PR DESCRIPTION
**Pull request by Weiyi-Feng**
fix load board issue

Tracked-On: #6691
Signed-off-by: Weiyi Feng <weiyix.feng@intel.com>

### CODE REVIEW

**Comment by acrnsi** [comment]
Can one of the admins verify this patch?

**Review by NanlinXie — APPROVED** [review]

**Comment by dbkinder** [comment]
@NanlinXie CI stuck?

**Comment by szhen11** [comment]
> @NanlinXie CI stuck?

Here CI is waiting for manually checking CI result before merge.

**Pull request merged**
